### ISSUE TITLE
Function paramters aren't displayed correctly when debugging on windows

### ISSUE THREAD

**Issue by AsafMah** [intellij-rust/intellij-rust#6676]
## Environment

* **IntelliJ Rust plugin version:** 0.3.140.3607-203-nightly
* **Rust toolchain version:** 1.51.0-nightly (e38fb306b 2021-01-14) x86_64-pc-windows-msvc
* **IDE name and version:** IntelliJ IDEA 2020.3.1 Ultimate Edition (IU-203.6682.168)
* **Operating system:** Windows 10 10.0
* **Macro expansion engine:** new
* **Name resolution engine:** old

## Problem description

Using intellij idea ultimate with debug, the debuggers variable tab doesn't render types like &str and Option correctly.
I might remember wrong, but I think the plugin did handle them before.
This is a new install so this might be the case.
The rust LLDB window also starts with an ominous error:
![image](https://user-images.githubusercontent.com/6424271/104739107-03a35200-574f-11eb-8cfc-47f270b834cd.png)

```
error: module importing failed: No module named 'renderers'
  File "temp.py", line 1, in <module>

error: 'jb_renderers_set_diagnostics_level' is not a valid command.
error: module importing failed: No module named 'renderers'
  File "temp.py", line 1, in <module>

error: 'jb_renderers_loaders_add' is not a valid command.

```

## Steps to reproduce
Intellij IDEA Ultimate + native debug + windows.
Create new project
start debugging

**Comment by MauriceKayser** [comment]
I have some more info:

```rust
fn main() {
    print_slice("12345678ABCDEFGH");
}

fn print_slice(s: &str) {
    println!("{}", s);
}
```

Place a breakpoint on the `println!` and the debugger will show the following output:

```
- s = {str}
  - data_ptr = { unsigned char * | 0x3837363534333231 } ""
  - length = { unsigned long long } 0x4847464544434241
```

There seems to be one dereferencing too much: Instead of reading from the slice reference, `data_ptr` seems to read the first 8 bytes of the slice data, and `length` seems to read the next 8 bytes of the slice data.

**Comment by ortem** [comment]
@AsafMah Thank you for the detailed description! I've managed to reproduce the problem.

The important thing is that the bug only affects function parameters, not local variables. So a possible hot-fix might be to bind the parameter value to a fresh local variable.

<img width="559" alt="Screenshot 2021-01-25 at 18 12 32" src="https://user-images.githubusercontent.com/4854600/105724508-fbaa9580-5f38-11eb-9359-0a84d740c4bf.png">

**Comment by ortem** [comment]
Looks like the bug affects function parameters which bind any heap values as well, for example, strings and vectors
<img width="622" alt="Screenshot 2021-01-25 at 18 37 37" src="https://user-images.githubusercontent.com/4854600/105727809-7fb24c80-5f3c-11eb-8dab-ed7eb7ccce45.png">

But does not affect function parameters which bind references
<img width="651" alt="Screenshot 2021-01-25 at 18 38 05" src="https://user-images.githubusercontent.com/4854600/105727815-817c1000-5f3c-11eb-85d1-b30f4f6e6b38.png">

**Comment by bkstein** [comment]
Just to remind: after almost 2 years, this bug is still not fixed. In my case, it results in CLion collecting data forever. I recently observed it with 50 GB of private bytes before I stopped it.

**Comment by ortem** [comment]
Fixed in CLion 2023.1 EAP 231.4840.

```Rust
fn print_vec(v: Vec<i32>) {
    println!("{:?}", v);
} // break

fn main() {
    print_vec(vec![1, 2, 3]);
}
```

![image](https://user-images.githubusercontent.com/4854600/213447483-bc5118bb-9561-4cbe-81e8-a1f596bc4e10.png)

Two important notes about this screenshot:
1. I put a breakpoint after the first statement to bypass https://github.com/intellij-rust/intellij-rust/issues/10008
2. I used `*v` (via Evaluate Expression or [Watches](https://www.jetbrains.com/help/clion/examining-suspended-program.html#watches)) because https://github.com/intellij-rust/intellij-rust/issues/9050 is not implemented yet. Parameters passed by value have a pointer-like representation in the debuginfo, when compiled via `x86_64-pc-windows-msvc` toolchain

Please note that IDEA Ultimate with Native Debug plugin 231.4840.379 does not contain the fix yet. It will be included in the next Native Debug plugin release, which will be published together with the next CLion EAP.
